openpype/hosts/maya/plugins/publish/extract_multiverse_usd.py

```diff
@@ -199,10 +201,10 @@ def process(self, instance):
             instance.data["representations"] = []
 
         representation = {
-            'name': 'usd',
-            'ext': 'usd',
+            'name': self.scene_type,
```

Comment:
@iLLiCiTiT would you feel it's wrong if `name` would always be `usd` no matter the extension? That way updating in the manager can be done even if file format changed in-between. Thoughts?